bin/azurite/wait

```diff
@@ -23,7 +23,7 @@ healthy() {
 }
 
 until healthy "$(curl --silent --max-time 2 "${azurite_endpoint}/")"; do
-  if (( "$(date +%s)" > "${DEADLINE}" )); then
```

Comment:
Non-blocking: Out of curiosity, is it reasonable to assume that all of the shell file changes here were purely the result of running the formatter and didn't require any manual changes on your part.